index.js

```diff
@@ -282,6 +286,10 @@ const registerEvents = () => {
 };
 
 export const config = plugin.config = {
+	validateOnlyWhenConfigured: {
```

Comment:
Alright. Let's just copy that: https://github.com/AtomLinter/linter-jscs/blob/6a3ceaade90d03b9952eb266da2e5b878b729e60/index.js#L24-L29

@fixe